figs/paper/paper.Rmd

```diff
@@ -0,0 +1,101 @@
+
+The workflow of this package is designed around this idea. Starting out with some dataset,
+
++ `specify()` allows the analyst to specify the variable, or relationship between variables, that they are interested in.
++ `hypothesize()` allows the analyst to declare the null hypothesis.
++ `generate()` allows the analyst to generate data reflecting the null hypothesis or using the bootstrap.
++ `calculate()` allows the analyst to calculate summary statistics, either from
+     * the observed data, to form the observed test statistic.
+     * data `generate()`d to reflect the null hypothesis, to form a randomization-based null distribution of test statistics.
+
+As such, the ultimate output of an infer pipeline using these four functions is generally an _observed statistic_ or _null distribution_ of test statistics. These four functions are thus supplemented with several utilities to visualize and extract value from their outputs.
+
++ `visualize()` plots the null distribution of test statistics.
+     * `shade_p_value()` situates the observed statistic in the null distribution, shading the region as or more extreme.
++ `get_p_value()` calculates a p-value via the juxtaposition of the test statistic and the null distribution.
+
+The workflow outlined above can also be used for constructing confidence intervals via bootstrapping with the omission of the `hypothesize()` step in the pipeline. The resulting bootstrap distribution can then be visualized with `visualize()`, the confidence interval region can be situated in the bootstrap distribution with `shade_confidence_interval()`, and the bounds of the confidence interval can be calculated with `get_confidence_interval()`.
+
+Beyond this, the `infer` package offers:
+
+* methods for inference using theory-based distributions
+* shorthand wrappers for common statistical tests using tidy data
+* model-fitting workflows to accommodate multiple explanatory variables
+
+# Comparison to Other Packages
+
+`broom` and `parameters` convert model objects to unified output formats, though they do not provide methods for fitting models, describing null distributions, performing bootstrapping, or calculating summary statistics from tabular data. `statsexpressions`, and adjacent packages in the `easystats` ecosystem, implement wrappers with consistent interfaces for theory-based hypothesis tests. At a higher level, the structure of each of these packages is defined by model types, where each model type has its own associated function and/or model object. In contrast, `infer` is structured around four functions, situating statistics and model types within a more abstracted grammar.^[This grammar follows from Allen Downey's "there is only one test" framework [@downey2011think].] `infer`'s initial release on the Comprehensive R Archive Network predated that of each of the packages mentioned above, with the exception of `broom`, a package situated in the same `tidymodels` ecosystem [@CRAN].
```

Comment:
Two comments here:

1. Seems like `mosaic` should be mentioned here. Something like: "mosaic is a package used to teach statistics by unifying summary statistics, visualization, and modeling with a consistent API built around R's formula interface. Included in the package is functionality to conduct resampling based inference.

2. I looked through Allen Downey's Think Stats book and couldn't find any mention of "There is only one test". As an alternative, his (revised) blogpost could work: http://allendowney.blogspot.com/2016/06/there-is-still-only-one-test.html

Reply:
You got it, thanks!